.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '37 23 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
```

Comment:
```suggestion
    name: analyze
```

Nit: We have a naming convention for actions of:

* Workflow names are camelcase - e.g. `CodeQL`
* Job names are lower with hyphens - e.g. `analyze`
* Steps are title case - e.g. `Checkout Repository`

<img width="451" alt="Screen Shot 2020-12-03 at 12 01 15 PM" src="https://user-images.githubusercontent.com/1049349/101081723-4de5c080-355f-11eb-8c63-5194fc30c387.png">